### PR TITLE
[MM-60825] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/mfa/setup/setup.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by TheInvincibleRalph**
#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/28501

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/mfa/setup/setup.tsx [hunk 1]
```diff
@@ -178,9 +178,12 @@ class Setup extends React.PureComponent<Props, State> {
                         />
                     </p>
                     <p>
-                        <FormattedMarkdownMessage
-                            id='mfa.setup.step2'
-                            defaultMessage='**Step 2: **Use Google Authenticator to scan this QR code, or manually type in the secret key'
+                        <FormattedMessage
+                            id='mfa.setup.step2_mfa'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/mfa/setup/setup.tsx:182]:
```suggestion
                            id='mfa.setup.step2_secret'
```

File: webapp/channels/src/components/mfa/setup/setup.tsx [hunk 2]
```diff
@@ -205,9 +208,12 @@ class Setup extends React.PureComponent<Props, State> {
                         </p>
                     </div>
                     <p>
-                        <FormattedMarkdownMessage
-                            id='mfa.setup.step3'
-                            defaultMessage='**Step 3: **Enter the code generated by Google Authenticator'
+                        <FormattedMessage
+                            id='mfa.setup.step3_mfa'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/mfa/setup/setup.tsx:212]:
```suggestion
                            id='mfa.setup.step3_code'
```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Just some nits

**Comment by TheInvincibleRalph** [comment]
Thanks for the review, @M-ZubairAhmed.

**Comment by TheInvincibleRalph** [comment]
Updated i18n. All test should pass now.

**Comment by TheInvincibleRalph** [comment]
I think I have made all requested changes. Or are there any other changes left for me to do? @M-ZubairAhmed

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by pvev — APPROVED** [review]
thanks for your contribution @TheInvincibleRalph 🎉

**Pull request merged**